zng/typetype.go

```diff
@@ -16,7 +16,7 @@ func EncodeType(s string) zcode.Bytes {
 
 func DecodeType(zv zcode.Bytes) (string, error) {
 	if zv == nil {
-		return "", ErrUnset
+		return "", nil
 	}
 	return string(zv), nil
```

Comment:
Nit: No need for the if statement; `return string(zv), nil` alone is sufficient.

Reply:
Hmm, just realized the zero-value of a type type should probably be "null".  I'll change it to that.  This stuff feels like it needs more clarification in the zson spec.